spring-websockets/pom.xml

```diff
@@ -37,6 +37,23 @@
 
     <build>
         <finalName>spring-websockets</finalName>
```

Comment:
Could you remove the plugin spring-boot-maven-plugin and add:      
      
        <spring-boot.repackage.skip>true</spring-boot.repackage.skip>
        
down in <properties> tag?

Reply:
sure